source/gui/addonGui.py

```diff
@@ -157,15 +157,21 @@ def __init__(self, parent):
 		AddonsDialog._instance = weakref.ref(self)
 		# Translators: The title of the Addons Dialog
 		title = _("Add-ons Manager")
-		wx.Dialog.__init__(self, parent, title=title)
+		# Translators: The title of the Addons Dialog when add-ons are disabled
+		titleWhenAddonsAreDisabled = _("Add-ons Manager (add-ons disabled)")
+		wx.Dialog.__init__(
+			self,
+			parent,
+			title=title if not globalVars.appArgs.disableAddons else titleWhenAddonsAreDisabled
+		)
 		DpiScalingHelperMixin.__init__(self, self.GetHandle())
 
 		mainSizer = wx.BoxSizer(wx.VERTICAL)
 		firstTextSizer = wx.BoxSizer(wx.VERTICAL)
 		listAndButtonsSizerHelper = guiHelper.BoxSizerHelper(self, sizer=wx.BoxSizer(wx.HORIZONTAL))
 		if globalVars.appArgs.disableAddons:
-			# Translators: A message in the add-ons manager shown when all add-ons are disabled.
-			label = _("All add-ons are currently disabled. To enable add-ons you must restart NVDA.")
+			# Translators: A message in the add-ons manager shown when add-ons are globally disabled.
+			label = _("NVDA was started with all add-ons disabled. You may modify the enabled / disabled state, and install or uninstall add-ons. Changes will not take effect after NVDA is restarted.")

```

Comment:
oops, it looks like I skipped an important word here, "until". Try the following:
```suggestion
			label = _("NVDA was started with all add-ons disabled. You may modify the enabled / disabled state, and install or uninstall add-ons. Changes will not take effect until after NVDA is restarted.")
```

Reply:
It's actually long enough that it would be nice to have it split up. I would split it at the sentence boundaries:

```suggestion
			label = _(
				"NVDA was started with all add-ons disabled. "
				"You may modify the enabled / disabled state, and install or uninstall add-ons. "
				"Changes will not take effect until after NVDA is restarted."
			)
```

Reply:
My bad, I didn't even read your suggestion, just copy and paste...